src/System.IO.FileSystem/ref/System.IO.FileSystem.cs

```diff
@@ -160,6 +160,8 @@ public sealed partial class FileInfo : System.IO.FileSystemInfo
         public void Decrypt() { }
         public void Encrypt() { }
         public void MoveTo(string destFileName) { }
+        public System.IO.FileInfo Replace(string destinationFileName, string destinationBackupFileName) { throw null; }
```

Comment:
>Replace [](start = 34, length = 7)

tiny nit, can you keep in alpha order - if we ever regenerate this file then it will diff nicer.